web/courses_test.go

```diff
@@ -214,8 +215,8 @@ func TestEnrollmentProcess(t *testing.T) {
 	}
 	// can't use: wantEnrollment.User.RemoveRemoteID()
 	wantEnrollment.User.RemoteIdentities = nil
-	if diff := cmp.Diff(pendingEnrollment, wantEnrollment, protocmp.Transform()); diff != "" {
-		t.Errorf("mismatch (-pendingEnrollment +wantEnrollment):\n%s", diff)
+	if diff := cmp.Diff(pendingEnrollment, wantEnrollment, cmpopts.IgnoreUnexported(pb.Enrollment{}, pb.User{}, pb.Course{})); diff != "" {
```

Comment:
Why replace the `protocmp.Transform()` here? You don't need the `IgnoreUnexported` here since all those types are protobuf types (from the `pb` package).